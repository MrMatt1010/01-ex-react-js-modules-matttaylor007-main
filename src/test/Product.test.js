import { render, cleanup } from "@testing-library/react";
import Product from "../components/Product";

afterEach(cleanup);

const description =
  "The sunglasses Ronnie Mac would wear if Ronnie Mac wore sunglasses.";
const image = "https://files.stripe.com/links/fl_test_HG5Y5oY5y2SlpQMclkQ0rjXm";
const name = "THE RONNIE MAC";

test("renders the Product component as expected", () => {
  const { asFragment } = render(
    <Product image={image} description={description} name={name} />
  );
  expect(asFragment()).toMatchSnapshot();
});
