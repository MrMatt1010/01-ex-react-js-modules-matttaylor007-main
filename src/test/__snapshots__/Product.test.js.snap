// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Product component as expected 1`] = `
<DocumentFragment>
  <li
    class="product-grid-item"
  >
    <img
      alt="THE RONNIE MAC"
      src="https://files.stripe.com/links/fl_test_HG5Y5oY5y2SlpQMclkQ0rjXm"
    />
    <h3>
      THE RONNIE MAC
    </h3>
    <p>
      The sunglasses Ronnie Mac would wear if Ronnie Mac wore sunglasses.
    </p>
    <button>
      Buy now 
    </button>
  </li>
</DocumentFragment>
`;
